bin/steps/pipenv

```diff
@@ -9,14 +9,14 @@ set -e
 if [[ -f Pipfile.lock ]]; then
     if [[ -f .heroku/python/Pipfile.lock.sha256 ]]; then
         if [[ $(openssl dgst -sha256 Pipfile.lock) == $(cat .heroku/python/Pipfile.lock.sha256) ]]; then
-            if [[ ! "$PIPENV_ALWAYS_INSTALL" ]]; then
+
+            # Don't skip installation of there are git deps.
+            if ! grep -q 'git' Pipfile.lock; then
```

Comment:
Do you want to grep for `git = ` instead of just `git`? This makes it a bit more likely it will just reference editable checkouts, rather than any of the 891 packages on PyPI which include the string `git` in their name (e.g. `django-logit`, `django-taggit`, anything referencing `digitalocean`...)

Reply:
yeah, i wasn't too worried about it, as the only opt-in here is proceeding to install

Reply:
problem is python might use `git = `, `git=`, `"git"  =  `

Reply:
I'd rather catch *all* genuine git deployments, than miss some, based on Pipfile syntax.